DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml

```diff
@@ -2,59 +2,73 @@
-                            </li>
-                        </ol>
-                </div>
-              @Html.HiddenFor(m => Model.Section.Name)
-              @Html.HiddenFor(m => Model.Section.SectionDataType)
-              @Html.HiddenFor(m => Model.OptionsSelected)
-              @Html.HiddenFor(m => Model.Section.PageNumber)
-              @Html.HiddenFor(m => Model.Section.SingleSelectOnly)
+        <form action="@Model.Section.PreviousPageUrl" method="post" id="backForm">
+            <div id="BackLink" class="govuk-breadcrumbs">
+                <a href="@Model.Section.PreviousPageUrl" class="govuk-back-link" id="filter-home">@(Model.Section.PageNumber == 1 ? "Home" : "Back")</a>
+                @*<button type="submit" class="govuk-back-link" id="filter-home">@(Model.Section.PageNumber == 1 ? "Home" : "Back")</button>*@
+            </div>
+            @Html.HiddenFor(m => Model.Section.Name)
+            @Html.HiddenFor(m => Model.Section.SectionDataType)
+            @Html.HiddenFor(m => Model.OptionsSelected)
+            @Html.HiddenFor(m => Model.Section.PageNumber)
+            @Html.HiddenFor(m => Model.Section.SingleSelectOnly)
         </form>
         <form action="@Model.Section.NextPageUrl" method="post" id="continueForm">
             <div class="active">
-                <h4>Step @Model.Section.PageNumber of @Model.Section.TotalNumberOfPages@if (@Model.Section.PageNumber > 1){<span> you have @Model.NumberOfMatches career matches</span>}</h4>
-                <h2>@Html.DisplayFor(model => Model.Section.Name)</h2>
-                @Html.Raw(Model.Section.Description)      
+                <span class="govuk-caption-xl">Step @Model.Section.PageNumber of @Model.Section.TotalNumberOfPages</span>
+                @*@if (@Model.Section.PageNumber > 1)
+                    {<span> you have @Model.NumberOfMatches career matches</span>}*@
+                <h1 class="govuk-heading-xl">@Html.DisplayFor(model => Model.Section.Name)</h1>
+                @Html.Raw(Model.Section.Description)
 
                 @Html.HiddenFor(m => Model.Section.Name)
                 @Html.HiddenFor(m => Model.Section.SectionDataType)
                 @Html.HiddenFor(m => Model.OptionsSelected)
                 @Html.HiddenFor(m => Model.Section.PageNumber)
                 @Html.HiddenFor(m => Model.Section.SingleSelectOnly)
 
-                <div class="filter-list">
-                    @if (Model.Section.Options != null && Model.Section.Options.Any())
-                    {
-                        for (int jj = 0; jj < Model.Section.Options.Count; jj++)
+                <div class="govuk-form-group">
+                    <fieldset class="govuk-fieldset" aria-describedby="qualifications-hint">
+                        <div class="govuk-hint"> Select all that apply. </div>
+                    </fieldset>
```

Comment:
Shouldn't this field set enclose all of the input elements? it should be closing just before the end of form tag

Reply:
This was updated